commitlint/plugins.ts

```diff
@@ -272,6 +272,24 @@ export abstract class Plugins {
         ];
     }
 
+    public static properRevertMessage(headerStr: string, bodyStr: string) {
+        let offence = false;
+
+        if (headerStr.match(/^[Rr]evert ".+"$/) !== null) {
+            // does msg have a body?
+            if (bodyStr !== "") {
+                let lines = bodyStr.split("\n");
+                offence =
+                    lines[0].match(/^This reverts commit [^ ]{40}\.$/) !==
```

Comment:
@realmarv 40? sorry, what is this magic number?

Reply:
The length of git commit hash

Reply:
it's not self-explanatory so you know what to do

Reply:
Done